### DISCUSSION TITLE
New version 2.3.0 and earlier versions

### DISCUSSION THREAD

**Discussion by joven1555** [hashlips-lab/nft-erc721-collection#239] [Q&A] [answered]
Hi there Liarco & team,

Just a quick question.  I see a new version was released 2 days ago and it made some small changes to the smart contract including lastOwnerAddress.  I'm just wondering how optimized you think the current version of this contract is at?  I've been holding off deploying as long as I could because I wanted the newest release and want to make sure we have as stable of a build as possible.

I'm going to move forward with 2.3.0 but I'm wondering how people have been fairing with previous versions until now.

Also I just want to say how AWESOME Liarco is!!!!

Thank you so much, sincerely,
Joven

**Comment by ClubPEC** [comment, marked as answer]
Hi Joven 

I used the 1st version to deploy my @club_pec contract. Till now we have 4 mints. It’s costing up to $15 per mint. 

Thanks 
PEC
